test/streamparse/test_dsl.py

```diff
@@ -1,95 +1,68 @@
 import unittest
 import logging
 
-from pystorm.spout import Spout
-from pystorm.bolt import Bolt
-
-from streamparse.dsl.topology import Topology, Grouping, TopologyError
-from streamparse.dsl.spec import Spec
+from streamparse.dsl.exceptions import TopologyError
+from streamparse.dsl.stream import Grouping
+from streamparse.dsl.topology import Topology
```

Comment:
In interest of "Flat is better than nested", we'll want to import a lot of these types into `streamparse.dsl` so that you only need one import line to get `Topology`, `Grouping`, and anything else necessary for the Topology specification.